.eslintrc

```diff
@@ -26,10 +26,10 @@
   "settings": {
     "ft-flow": {
       "onlyFilesWithFlowAnnotation": true
-    },
+    }
   },
   "env": {
-    "node": true,
+    "node": false,
```

Comment:
Why?

Reply:
it was temporary measure to ask eslint to please shut up until I got a good picture of what I needed to change :-)
I can revert that to `true` along with removing those last __filename and __dirname from bin/web-ext.js.